kyo-core/shared/src/main/scala/kyo/resources.scala

```diff
@@ -20,9 +20,13 @@ object Resources:
                 }
         }
 
+    def acquireRelease[T](acquire: => T < IOs)(release: T => Unit < IOs): T < Resources =
+        acquire.map { resource =>
```

Comment:
When I define a method that takes a by-name `T < IOs`, I normally also add an `IOs` suspension since it's easy for users to forget to add it:

```scala
IOs {
   acquire.map { resource =>
      ...
   }  
}
```